server/services/property.service.js

```diff
@@ -53,12 +53,39 @@ export const updateProperty = async (updatedProperty) => {
     throw new ErrorHandler(httpStatus.INTERNAL_SERVER_ERROR, 'Internal Server Error', error);
   });
 
+  let { units, availableUnits } = property;
+
+  if (updatedProperty.units) {
+    const initialUnits = property.units;
+    const updatedunits = updatedProperty.units;
```

Comment:
When writing compound words in Javascript, always ensure you use camelCasing:

```suggestion
    const updatedUnits = updatedProperty.units;
```